src/main/java/com/palantir/gradle/versions/VersionsLockPlugin.java

```diff
@@ -1044,6 +1062,69 @@ private static LockedConfigurations computeConfigurationsToLock(Project project,
                 .collect(Collectors.toList());
     }
 
+    private static List<DependencyConstraint> constructPublishableConstraintsFromLocalProjects(
+            Project rootProject, Project currentProject, DependencyConstraintCreator constraintCreator) {
+        // Include all other libraries published from the same repository
+        return rootProject.getAllprojects().stream()
```

Comment:
Optionally we can also do `currentProject.getRootProject()` and drop the `rootProject` param.